hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/compact/schema/SchemaReplicationStatus.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.serialization.impl.compact.schema;
+
+/**
+ * Describes the status of the schema replication for a particular
+ * schema id.
+ */
+public enum SchemaReplicationStatus {
+    /**
+     * The schema is available in the in-memory registry of the member,
+     * persisted to the HotRestart (if enabled), and replicated in the
+     * WAN clusters (if enabled, only for the initiator node).
+     */
```

Comment:
Note to myself, so that I can fix it while working on review comments: The last part about WAN is not up to date